.github/workflows/long-test.yml

```diff
@@ -149,3 +149,171 @@ jobs:
             build/workspace/*/err
             build/workspace/*.ledger/*
           if-no-files-found: ignore
+
+  long-e2e-debug:
```

Comment:
Describing the intent for these jobs, either here in comment or in the README.md, is probably a good idea